agreement/msgp_gen_test.go

```diff
@@ -1,3 +1,4 @@
+//go:build !skip_msgp_testing
```

Comment:
Since this file is generated, to avoid having to reapply this change every time these files are generated this would have to be added here too: https://github.com/algorand/msgp/blob/a756cc98119ad08a8fb2aa6ef9a4a844016644b5/printer/print.go#L147-L148

Reply:
ACK, will update.

Reply:
https://github.com/algorand/msgp/pull/15